packages/cli/src/types.ts

```diff
@@ -62,18 +62,26 @@ export type User = {
     updatedAt: number;
   };
   name?: string;
+  limited?: boolean;
 };
 
-export type Team = {
+export interface Team {
   id: string;
-  avatar?: string;
+  avatar?: string | null;
   billing: Billing;
   created: string;
   creatorId: string;
   membership: { uid: string; role: 'MEMBER' | 'OWNER'; created: number };
   name: string;
   slug: string;
-};
+  limited?: boolean;
+  saml?: {
+    enforced: boolean;
```

Comment:
Should `enforced` be optional too?

Reply:
No, it's always present if the `saml` object exists.